go.mod

```diff
@@ -102,6 +102,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.6.4
 	sigs.k8s.io/controller-tools v0.4.0
 	sigs.k8s.io/kubefed v0.4.0
+	kubesphere.io/ks-client v0.0.0
```

Comment:
I suggest we follow conventions like K8s, using the name kubesphere.io/client-go in case there are other implementations like Java, Python someday.